src/Entity/Adherent.php

```diff
@@ -328,6 +328,46 @@ class Adherent implements UserInterface, UserEntityInterface, GeoPointInterface,
      */
     private $ideas;
 
+    /**
+     * @var Media|null
+     *
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Media", cascade={"persist"})
+     * @ORM\JoinColumns({
+     *     @ORM\JoinColumn(name="media_id", referencedColumnName="id", nullable=true)
+     * })
```

Comment:
Can't you declare `@ORM\JoinColumn` directly ?

Reply:
i need to set nullable at true.